epxresults/job.py

```diff
@@ -567,6 +567,7 @@ def _set_snapshot_map(self) -> Dict[dt.date, Snapshot]:
         self._snapshot_map = {}
         for snapshot_file in self._parse_snapshots():
             snapshot = Snapshot(PATH_TO_SNAPSHOT=os.path.join(self.path_to_job,
+                                                              'OUT',
                                                               snapshot_file))
```

Comment:
I forget what the relationship is between the snapshot files in the `RUN*` dirs vs the `OUT` dir but trust that you've got this down.